applications/luci-app-statistics/htdocs/luci-static/resources/statistics/rrdtool/definitions/dsl.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
Can you please follow the syntax of the other modules?
If it's all so compact, then you can't really track changes.
So please have a look at how the other modules have done it.